clang-tools-extra/test/clang-doc/invalid-file-error.test

```diff
@@ -0,0 +1,4 @@
+// RUN: not clang-doc %S/Inputs/basic-project/src/Circle.cpp -output=/root/docs 2>&1 | FileCheck %s
+
+// CHECK: clang-doc error: Failed to create directory
```

Comment:
CI says this check is not passing, so do check why. I'm guessing its the file path to Circle.cpp, so it may start working better if you switch to %s.